dolphin/drivers/dellemc_storage/vmax.py

```diff
@@ -0,0 +1,149 @@
+        self.storage_id = storage_id
+
+    @staticmethod
+    def get_storage_registry():
+        register_info = {
+            "server_ip": "192.168.20.158",
+            "port": 8443,
+            "username": "unisphere-user",
+            "password": "secret-pass",
+            "u4v_version":'90',
+            "verify": "unisphere-ssl-cakey-file"
+        }
+        LOG.info('get_storage_registry()')
+        return register_info
+
+    def register_storage(self, context, register_info):
+        try:
+            u4v_version = register_info['u4v_version']
+            server_ip = register_info['server_ip']
+            port = register_info['port']
+            verify = register_info['verify']
+            username = register_info['username']
+            password = register_info['password']
+
+            # Initialise PyU4V connection to Unisphere
+            conn = PyU4V.U4VConn(
+                u4v_version=u4v_version, server_ip=server_ip, port=port,
+                verify=verify, username=username, password=password)
+
+            # Get the Unisphere version
+            version = conn.common.get_uni_version()
+
+            # Retrieve a list of arrays managed by your instance of Unisphere
+            array_list = conn.common.get_array_list()
+
+            # Close the session
+            conn.close_session()
+
+        except:
+            print('Failed to connect to storage')
+
+        else:
+            # Update storage_id's that this driver manages
+            if VMAXStorageDriver.storage_ids:
+                if self.storage_id in VMAXStorageDriver.storage_ids:
+                    LOG.info('Storage is already registered!')
+                else:
+                    VMAXStorageDriver.storage_ids.append(self.storage_id)
+            else:
+                VMAXStorageDriver.storage_ids = [self.storage_id]
+        
+        print('register_storage()')
+        return VMAXStorageDriver.storage_ids
+
+    def get_storage(self, context):
+        try:
+            u4v_version = register_info['u4v_version']
+            server_ip = register_info['server_ip']
+            port = register_info['port']
+            verify = register_info['verify']
+            username = register_info['username']
+            password = register_info['password']
+
+            # Initialise PyU4V connection to Unisphere
+            conn = PyU4V.U4VConn(
+                u4v_version=u4v_version, server_ip=server_ip, port=port,
+                verify=verify, username=username, password=password)
+
+
+            # Get the Unisphere version
+            version = conn.common.get_uni_version()
+
+            # Retrieve a list of arrays managed by your instance of Unisphere
+            array_list = conn.common.get_array_list()
+
+            # Output results to screen
+            LOG.info('version is: {ver}'.format(ver=version[0]))
+            LOG.info('This instance of Unisphere instance manages the following arrays: '
```

Comment:
Just suggestion: It will remove multi-time use of 'arr_list'
LOG.info('This instance of Unisphere instance manages the following arrays: '
                '{arr_list}'.format(arr_list=array_list))

LOG.info('This instance of Unisphere instance manages the following arrays: '
                **'{0}'.format(array_list))**

And also please format the lines based on python conventions.

Reply:
Removed.